config/locales/es.yml

```diff
@@ -832,6 +832,7 @@ es:
       case: "Caso"
       child: "Caso"
       incident: "Incidente"
+      kpi: "KPI"
```

Comment:
The only locale yml you need to update is 'en.yml'.   We populate the others by loading en.yml into Transifex and then let other folks translate it for us.